projects/driver_controls/src/stm32f0xx/ads1015.c

```diff
@@ -173,11 +176,14 @@ StatusCode ads1015_read_converted(Ads1015Storage *storage, Ads1015Channel channe
     return status_code(STATUS_CODE_INVALID_ARGS);
   }
   int16_t raw_reading = ADS1015_READ_UNSUCCESSFUL;
-  status_ok_or_return(ads1015_read_raw(storage, channel, &raw_reading));
+  ads1015_read_raw(storage, channel, &raw_reading);
```

Comment:
I think you want to be checking the status code here. I'm not sure why you removed it.

From the user's perspective, if they receive a non-OK status code, they're just going to ignore the returned value. There's no point in explicitly setting it again.

Reply:
Thats because if I return there, I would fail to read int max into the disabled channel reading. read raw would fail if the channel is disabled but i still want to read converted.

Reply:
From the user's perspective, that behavior is unnecessary. Why would they care what the reading is if they get a non-OK status code?